torch/csrc/jit/runtime/register_prim_ops_fulljit.cpp

```diff
@@ -137,24 +137,33 @@ RegisterOperators reg(
      // the whole gradients in every tensor of the Autograd graph with
      // create_graph=True so we use aliasAnalysisConservative for these two OPs
      Operator(
-         "aten::backward.TensorList(Tensor[] tensors, Tensor?[]? grad_tensors=None, bool? retain_graph=None, bool create_graph=False) -> ()",
+         "aten::backward.TensorList(Tensor[] tensors, Tensor?[]? grad_tensors=None, bool? retain_graph=None, bool create_graph=False, Tensor?[]? inputs=None) -> ()",
          [](Stack* stack) {
-           bool create_graph = pop(stack).toBool();
-           auto retain_graph = pop(stack).toOptional<bool>();
            auto grad_tensors = pop(stack);
+           auto retain_graph = pop(stack).toOptional<bool>();
```

Comment:
Does that mean that the current code is wrong and is not reading from the stack properly?

Reply:
Ah no, I see what the code is doing now.